src/framework/shortcuts/internal/shortcutsregister.cpp

```diff
@@ -32,6 +34,7 @@
 using namespace mu::shortcuts;
 using namespace mu::framework;
 using namespace mu::async;
+using namespace mu::ui;
```

Comment:
Are the changes in this file fix-ups for previous changes, or are they left-overs of uncommitted experiments?